src/test/java/com/training/web/tests/flipkart/FlipkartTest.java

```diff
@@ -67,4 +68,80 @@ public void test() throws WebDriverException {
          */
         driver.close();
     }
+
+    @Test
+    public void testCartAddition() {
+
+        Integer[] products = {2, 3};
+        Integer[] size = {8, 8};
+
+        /**
+         * System Property for Chrome Driver
+         */
+        System.setProperty("webdriver.chrome.driver", "package/resources/chromedriver.exe");
+
+        /**
+         * Instantiate a ChromeDriver class.
+         */
+        ChromeOptions options = new ChromeOptions().addArguments("incognito");
+        DesiredCapabilities capabilities = DesiredCapabilities.chrome();
+        capabilities.setCapability(ChromeOptions.CAPABILITY, options);
+        WebDriver driver = new ChromeDriver(capabilities);
+
+        /**
+         * Launch Website and maximise
+         */
+        driver.navigate().to("https://www.flipkart.com/");
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+        String parentWindow = driver.getWindowHandle();
+
+        /**
+         * Close popup and search for shoes
+         */
+        ResultPage resultPage = new FlipkartHomePage(driver).closePopup().sendKeysToSearchBox("shoes")
+                .clickSearch().sortLowToHigh();
+
+        /**
+         * Select and add to cart item 2 and 3
+         */
+        ArrayList<String> productNames = new ArrayList<>();
+        ArrayList<Integer> priceList = new ArrayList<>();
+        resultPage.getProductsList();
```

Comment:
not sure you understood this change. 
getProductsList() should return a list - which would be used here in the test class. This would avoid the use of global dependent variable